### ISSUE TITLE
Version 9.1.1 of nativescript-list-view does not work.

### ISSUE THREAD

**Issue by jcassidyav** [telerik/nativescript-ui-feedback#1530]
√ Component nativescript has 8.0.2 version and is up to date.
√ Component @nativescript/core has 8.0.7 version and is up to date.
√ Component @nativescript/ios has 8.0.0 version and is up to date.
√ Component @nativescript/android has 8.0.0 version and is up to date.

compiling a sample app with Version 9.1.1 of nativescript-list-view
gives 

```
ERROR in ./node_modules/nativescript-ui-listview/ui-listview.android.js 570:0
Module parse failed: The keyword 'interface' is reserved (570:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     ListViewDataSourceAdapterClass = ListViewDataSourceAdapter;
| }
> interface ListViewItemClickListener {
|     new (owner: WeakRef<RadListView>): com.telerik.widget.list.RadListView.ItemClickListener;
| }
 @ ./node_modules/nativescript-ui-listview/__ivy_ngcc__/angular/fesm2015/nativescript-ui-listview-angular.js 4:0-164 31:44-64 44:21-46 48:34-41 52:21-51 56:34-41 60:21-54 65:35-42 69:21-48 74:35-42 78:21-47 86:35-42 90:21-48 95:35-42 249:31-38 253:45-52 302:27-34 650:37-48 651:46-66 652:44-62 653:49-72 654:39-52
 @ ./src/app/app.module.ts 7:0-80 14:74-102
 @ ./src/main.ts 4:0-45 6:69-78
```

Sample angular app created from template:

[testlist.zip](https://github.com/ProgressNS/nativescript-ui-feedback/files/6637462/testlist.zip)

**Comment by timdoege** [comment]
Yes, I just tried 9.1.1 and got
```

CONSOLE LOG: Loading inspector modules...
CONSOLE LOG: Finished loading inspector modules.
***** Fatal JavaScript exception - application has been terminated. *****
NativeScript encountered a fatal error: Uncaught ReferenceError: NativeClass is not defined
at
./node_modules/nativescript-ui-listview/ui-listview.ios.js(file: node_modules/nativescript-ui-listview/ui-listview.ios.js:511:0)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./node_modules/nativescript-ui-listview/angular/fesm2015/nativescript-ui-listview-angular.js(file:///app/vendor.js:221189:82)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./src/app/core/core.module.ts(file:///app/bundle.js:1189:91)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./src/app/app.module.ts(file:///app/bundle.js:748:75)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./src/main.ts(file:///app/bundle.js:31:73)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at (file: src/webpack/bootstrap:36:0)
at __webpack_require__.O(file: src/webpack/runtime/chunk loaded:23:0)
at __webpack_require__.x(file: src/webpack/bootstrap:37:0)
at __webpack_require__.x(file: src/webpack/runtime/startup chunk dependencies:4:0)
at (file: src/webpack/startup:2:0<…>
(CoreFoundation) *** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: Uncaught ReferenceError: NativeClass is not defined
at
./node_modules/nativescript-ui-listview/ui-listview.ios.js(file: node_modules/nativescript-ui-listview/ui-listview.ios.js:511:0)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./node_modules/nativescript-ui-listview/angular/fesm2015/nativescript-ui-listview-angular.js(file:///app/vendor.js:221189:82)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./src/app/core/core.module.ts(file:///app/bundle.js:1189:91)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./src/app/app.module.ts(file:///app/bundle.js:748:75)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at ./src/main.ts(file:///app/bundle.js:31:73)
at __webpack_require__(file: src/webpack/bootstrap:19:0)
at (file: src/webpack/bootstrap:36:0)
at __webpack_require__.O(file: src/webpack/runtime/chunk loaded:23:0)
at __webpack_require__.x(file: src/webpack/bootstrap:37:0)
at __webpack_require__.x(file:///app/bundle.j<…>
CONSOLE LOG: Loading inspector modules...
CONSOLE LOG: Finished loading inspector modules.
***** Fatal JavaScript exception - application has been terminated. *****
```

(on NS 8 + Angular 11)

- rolled back to 9.1.0 for now.

**Comment by jcassidyav** [comment]
I think the processing of the NativeClass stuff didn't happen ( ie. ts-patch install wasn't executed ) when it was built but that is just a guess.

**Comment by jcassidyav** [comment]
9.1.2 is also broken, it is missing the com.telerik.widget.list.Utils class for android.

**Comment by timdoege** [comment]
Yes, on 9.1.2 I get
```
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onCreateView failed
System.err: TypeError: Cannot read property 'createSource' of undefined
System.err: 
System.err: StackTrace:
System.err: loadData(file: node_modules/nativescript-ui-listview/ui-listview.android.js:2038:0)
System.err: 	at createNativeView(file: node_modules/nativescript-ui-listview/ui-listview.android.js:1178:0)
System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:554:0)
System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/layouts/layout-base-common.js:101:0)
System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/content-view/index.js:65:0)
System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/layouts/layout-base-common.js:101:0)
System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/layouts/layout-base-common.js:101:0)
System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/content-view/index.js:65:0)
System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/page/page-common.js:106:0)
System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
System.err: 	at _addViewCore(file: node_modules/@nativescript/core/ui/core/view-base/index.js:450:0)
System.err: 	at _addView(file: node_modules/@nativescript/core/ui/core/view-base/index.js:439:0)
System.err: 	at onCreateView(file: node_modules/@nativescript/core/ui/frame/index.android.js:739:0)
System.err: 	at onCreateView(file: node_modules/@nativescript/core/ui/frame/fragment.android.js:25:0)
System.err: 	at com.tns.Runtime.callJSMethodNative(Native Method)
System.err: 	at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err: 	at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err: 	at com.tns.FragmentClass.onCreateView(FragmentClass.java:55)
System.err: 	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2699)
System.err: 	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:320)
System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1199)
System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
System.err: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
System.err: 	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
System.err: 	at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
System.err: 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
System.err: 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
System.err: 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1861)
System.err: 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2641)
System.err: 	at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2589)
System.err: 	at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2723)
System.err: 	at androidx.fragment.app.FragmentStateManager.activityCreated(FragmentStateManager.java:346)
System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1200)
System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
System.err: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
System.err: 	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
System.err: 	at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
System.err: 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
System.err: 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
System.err: 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1861)
System.err: 	at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:413)
System.err: 	at android.os.Handler.handleCallback(Handler.java:883)
System.err: 	at android.os.Handler.dispatchMessage(Handler.java:100)
System.err: 	at android.os.Looper.loop(Looper.java:214)
System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
System.err: 	at java.lang.reflect.Method.invoke(Native Method)
System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by SmailHammour** [comment]
Same here, 9.1.2;
```
JS: ERROR TypeError: Cannot read property 'createSource' of undefined
JS: ERROR TypeError: Cannot read property 'createSource' of undefined
JS: ERROR UnsubscriptionError: 1 errors occurred during unsubscription:
JS: 1) TypeError: Cannot read property 'createSource' of undefined
JS: ERROR TypeError: Cannot read property 'createSource' of undefined
```

Reverting back to 9.1.0 seems to fix the issue.

**Comment by jelledijkstra97** [comment]
Same issue here. 9.1.0 works fine.

**Comment by Lengo46** [comment]
+1

**Comment by PabloChiste** [comment]
Same here, 9.1.0, 9.1.1 and 9.1.2

always get JS: ERROR TypeError: Cannot read property 'createSource' of undefined

**Comment by MrDrk94** [comment]
> Yes, on 9.1.2 I get
> 
> ```
> System.err: An uncaught Exception occurred on "main" thread.
> System.err: Calling js method onCreateView failed
> System.err: TypeError: Cannot read property 'createSource' of undefined
> System.err: 
> System.err: StackTrace:
> System.err: loadData(file: node_modules/nativescript-ui-listview/ui-listview.android.js:2038:0)
> System.err: 	at createNativeView(file: node_modules/nativescript-ui-listview/ui-listview.android.js:1178:0)
> System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:554:0)
> System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
> System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/layouts/layout-base-common.js:101:0)
> System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
> System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
> System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
> System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/content-view/index.js:65:0)
> System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
> System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
> System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
> System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/layouts/layout-base-common.js:101:0)
> System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
> System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
> System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
> System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/layouts/layout-base-common.js:101:0)
> System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
> System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
> System.err: 	at (file: node_modules/@nativescript/core/ui/core/view-base/index.js:600:0)
> System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/content-view/index.js:65:0)
> System.err: 	at eachChildView(file: node_modules/@nativescript/core/ui/page/page-common.js:106:0)
> System.err: 	at eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
> System.err: 	at _setupUI(file: node_modules/@nativescript/core/ui/core/view-base/index.js:599:0)
> System.err: 	at _addViewCore(file: node_modules/@nativescript/core/ui/core/view-base/index.js:450:0)
> System.err: 	at _addView(file: node_modules/@nativescript/core/ui/core/view-base/index.js:439:0)
> System.err: 	at onCreateView(file: node_modules/@nativescript/core/ui/frame/index.android.js:739:0)
> System.err: 	at onCreateView(file: node_modules/@nativescript/core/ui/frame/fragment.android.js:25:0)
> System.err: 	at com.tns.Runtime.callJSMethodNative(Native Method)
> System.err: 	at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
> System.err: 	at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
> System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1175)
> System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1153)
> System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1149)
> System.err: 	at com.tns.FragmentClass.onCreateView(FragmentClass.java:55)
> System.err: 	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2699)
> System.err: 	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:320)
> System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1199)
> System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
> System.err: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
> System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
> System.err: 	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
> System.err: 	at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
> System.err: 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
> System.err: 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
> System.err: 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1861)
> System.err: 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2641)
> System.err: 	at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2589)
> System.err: 	at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2723)
> System.err: 	at androidx.fragment.app.FragmentStateManager.activityCreated(FragmentStateManager.java:346)
> System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1200)
> System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
> System.err: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
> System.err: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
> System.err: 	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
> System.err: 	at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
> System.err: 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
> System.err: 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
> System.err: 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1861)
> System.err: 	at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:413)
> System.err: 	at android.os.Handler.handleCallback(Handler.java:883)
> System.err: 	at android.os.Handler.dispatchMessage(Handler.java:100)
> System.err: 	at android.os.Looper.loop(Looper.java:214)
> System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
> System.err: 	at java.lang.reflect.Method.invoke(Native Method)
> System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
> System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
> ```

I got same error with 9.1.0

**Comment by SantiagoGuerra** [comment]
Same here. Version 9.1.0

```
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onCreateView failed
System.err: TypeError: Cannot read property 'createSource' of undefined
System.err:
System.err: StackTrace:
System.err: loadData(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:217381:60)
System.err:     at createNativeView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:216521:14)
System.err:     at _setupUI(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120686:31)
System.err:     at (file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120730:19)
System.err:     at eachChildView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:131024:28)
System.err:     at eachChild(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:123019:14)
System.err:     at _setupUI(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120729:14)
System.err:     at (file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120730:19)
System.err:     at eachChildView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:118161:13)
System.err:     at eachChild(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:123019:14)
System.err:     at _setupUI(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120729:14)
System.err:     at (file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120730:19)
System.err:     at eachChildView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:118161:13)
System.err:     at eachChild(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:123019:14)
System.err:     at _setupUI(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120729:14)
System.err:     at (file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120730:19)
System.err:     at eachChildView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:131024:28)
System.err:     at eachChild(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:123019:14)
System.err:     at _setupUI(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120729:14)
System.err:     at (file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120730:19)
System.err:     at eachChildView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:131024:28)
System.err:     at eachChild(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:123019:14)
System.err:     at _setupUI(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120729:14)
System.err:     at (file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120730:19)
System.err:     at eachChildView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:118161:13)
System.err:     at eachChildView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:132962:15)
System.err:     at eachChild(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:123019:14)
System.err:     at _setupUI(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120729:14)
System.err:     at _addViewCore(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120582:18)
System.err:     at _addView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:120571:14)
System.err:     at onCreateView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:127065:19)
System.err:     at onCreateView(file:///data/user/0/com.sharedspace.app/files/app/vendor.js:124890:40)
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.FragmentClass.onCreateView(FragmentClass.java:55)
System.err:     at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2699)
System.err:     at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:320)
System.err:     at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1199)
System.err:     at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
System.err:     at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
System.err:     at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
System.err:     at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
System.err:     at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
System.err:     at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
System.err:     at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
System.err:     at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1861)
System.err:     at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:413)
System.err:     at android.os.Handler.handleCallback(Handler.java:938)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:99)
System.err:     at android.os.Looper.loop(Looper.java:223)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:7656)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by amine2050** [comment]
Using Version 9.1.0, I am suddenly getting the error `ERROR TypeError: Cannot read property 'createSource' of undefined` on a previously working project (no source changes), just after using `tns clean` which deletes the package-lock.json file (recreated on build).

Restoring the old package-lock.json fixed the problem.

Here are the good and the bad package-lock.json files
[Good package-lock.json](https://github.com/ProgressNS/nativescript-ui-feedback/files/6701002/Good_package-lock.json.txt)
[Bad package-lock.json](https://github.com/ProgressNS/nativescript-ui-feedback/files/6701003/Bad_package-lock.json.txt)

**Comment by jcassidyav** [comment]
> Using Version 9.1.0, I am suddenly getting the error `ERROR TypeError: Cannot read property 'createSource' of undefined` on a previously working project (no source changes), just after using `tns clean` which deletes the package-lock.json file (recreated on build).
> 
> Restoring the old package-lock.json fixed the problem.

if you remove the package-lock.json npm will fetch the latest version matching what you have in package.json, in this case it has fetched 9.1.2  in your package.json you need to fix the version to the last known working one, 

```"nativescript-ui-listview": "9.1.0",```

**Comment by update-switzerland** [comment]
I have the same experience as @amine2050.
I have explicitly set 9.1.0 in my package.json and after a clean it does download this exact version. I still get the error, though.

**Comment by MrDrk94** [comment]
Things that I found until now is: 
As @amine2050  's said:

> using `ns clean` removes package-lock.json

 and as @jcassidyav mentioned :

>  if you remove the package-lock.json npm will fetch the latest version matching what you have in package.json

I faced this problem suddenly because `"nativescript-ui-listview": "^9.1.0"` in my package.json file. I changed this to `"nativescript-ui-listview": "9.1.0"` be careful I removed '^' sign. This means "Do not upgrade to latest version, just use exact version I determine". I hope this can fix things. 

However version 9.1.2 doesn't work and I could not figured out this.
